frontend/src/metabase/lib/formatting.js

```diff
@@ -361,6 +361,11 @@ export function slugify(name: string) {
     return name && name.toLowerCase().replace(/[^a-z0-9_]/g, "_");
 }
 
+// Adds commas and spaces between array items and replaces the comma between two last items with "and"
+export function formatListOfItems(items: []): string {
+    return [items.slice(0, -1).join(', '), items.slice(-1)[0]].join(items.length < 2 ? '' : ' and ');
+}
```

Comment:
since all the code that's using this is commented out we may as well just remove the function for now and save it for a PR that actually uses that

failing that, we should write tests for it. I don't see any tests :eyes:

Reply:
yup we can remove this function

Reply:
+1

Reply:
removed